File: lab8/src/main/java/Pump.java
public class Pump extends Good {

    private double diametr;

    public Pump(final GoodType goodType, final GoodName name, final int price, final String material,
                final double weight, final String manufucturer,
                final String color, final int amount, final double diametr) {

        super(goodType, name, price, material, weight, manufucturer, color, amount);
        setDiametr(diametr);
    }

    public String getHeaders() {
        return super.getHeaders() + "," + "diametr";
    }

    public String toCVS() {
        return super.toCVS() + "," + getDiametr();
    }

    public final double getDiametr() {
        return diametr;
    }

    public final void setDiametr(final double diametr) {
        this.diametr = diametr;
    }
}

File: Python lab2/goods/Pump.py
from goods.Good import *
from enums.GoodType import *
from enums.GoodName import *


class Pump(Good):
    goods_type = GoodType.FORANY
    goods_name = GoodName.PUMP

    def __init__(self, diametr, name, price, material, weight, manufacturer, color, amount):
        self.diametr = diametr
        Good.__init__(self, name, price, material, weight, manufacturer, color, amount)

    def __str__(self):
        return "Name: " + str(self.name) + " price: " + str(self.price) + " diametr:" + str(self.diametr)
